website/scatter3.Rmd

```diff
@@ -114,3 +142,5 @@ ggplot(data = mpg) +
 ```
 
 Mapping the same variable to two aesthetics is sometimes useful to further emphasize this variable.
+
```

Comment:
Can you please add captions before these pointers here and elsewhere?

Reply:
What do you mean by captions?

Reply:
captions -> subsections, starting with `##`